src/classes/CON_PreferredPhone_TEST.cls

```diff
@@ -36,33 +36,32 @@
 */
 @isTest
 private class CON_PreferredPhone_TEST {
-    
-    @testSetup
-    static void dataSetup() {
-        
-        UTIL_CustomSettingsFacade.getSettingsForTests(
-            new Hierarchy_Settings__c(Disable_Preferred_Phone_Enforcement__c = False)
+
+    private static Hierarchy_Settings__c enablePreferredPhone() {
+        return UTIL_CustomSettingsFacade.getSettingsForTests(
+            new Hierarchy_Settings__c(
+                Disable_Preferred_Phone_Enforcement__c = false, 
```

Comment:
Sorry for some of these spacing/formatting changes in this file. The main change is that we have two new methods: enablePreferredPhone() and disablePreferredPhone() that sets the test up as expected.

There's no longer a need to modify the TDTM record.